lib/improver/lapse_rate.py

```diff
@@ -257,11 +257,11 @@ def _calc_lapse_rate(self, temperature, orography):
         to find the local lapse rate.
```

Comment:
The `SaveNeighbourhood` class should be updated so that `buffer` is defined as a `numpy.ndarray`. Ideally, I think that the `allbuffers` argument should also be defined. I think this argument is a `numpy.ndarray`.